src/examples/zoom-and-scale.html

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<body>
+  <script src="http://d3js.org/d3.v3.min.js"></script>
+  <script src="http://d3js.org/topojson.v1.min.js"></script>
+  <!-- I recommend you host this file on your own, since this will change without warning -->
+  <script src="/src/rel/datamaps.world.js?v=1"></script>
```

Comment:
You are not using the compiled version of the library. So no `vector-effect: non-scaling-stroke`.